pkg/cli/environment.go

```diff
@@ -213,6 +218,23 @@ func envCSV(name string) (ls []string) {
 	return
 }
 
+func envColorMode() string {
+	// Check NO_COLOR environment variable first (standard)
+	if v, ok := os.LookupEnv("NO_COLOR"); ok && v != "" {
+		return "never"
+	}
+	// Check HELM_COLOR environment variable
+	if v, ok := os.LookupEnv("HELM_COLOR"); ok {
+		v = strings.ToLower(v)
+		switch v {
+		case "never", "auto", "always":
+			return v
+		}
+	}
+	// Default to never (disabled) until more commands support color
+	return "never"
```

Comment:
I differ in opinion for @gjenkins8. I don't see any harm in using auto by default, especially as an initial v4 release of this feature. If we default to `never` prior to locking v4, we'll be stuck with that default until v5. Let's do it now and ensure that users don't shoot themselves in the foot by coding around a default we'd like to later change.

Reply:
I agree with this approach. Could we discuss further on this @gjenkins8 ?

Reply:
sure, lets default to "auto" then